src/test/java/org/apache/mesos/testutils/NetworkTestUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.testutils;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * This class provides utilities useful for all tests which may have Network related needs.
+ */
+public class NetworkTestUtils {
```

Comment:
I'm a bit skeptical about having a `NetworkTestUtils` for a single use-case. Not completely against it, but keeping our ongoing focus on code-quality and simplicity in mind, just wanted to keep code-base real tight and only generalize something when it's required at more than 2 places.